### ISSUE TITLE
Feature Request - Sleeper league median

### ISSUE THREAD

**Issue by tjhall9** [mikemountain/fantasy-football-scoreboard#37]
A lot of leagues are playing with league median now, would love to have this on screen in small text between the logos.

Might try to attempt this myself but haven't done much API work yet so no promises.

**Comment by mikemountain** [comment]
Yeah, this should be doable. Will have to see if sleeper's API has that median info anywhere, but if it doesn't it'd be a quick calculation. Can probably replace the "Week X" info with a median that gets updated so the small score adjustments don't have to disappear
